odl/contrib/datasets/ct/mayo.py

```diff
@@ -0,0 +1,270 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Tomographic datasets from Mayo Clinic.
```

Comment:
the

Reply:
Actually it is a name and I'm quite sure it should be written like this, did some googling and it seems to be proper style

Reply:
Oh, you're right, of course.